spec/factories/lti_services.rb

```diff
@@ -0,0 +1,7 @@
+FactoryBot.define do
+  factory :lti_service do
+    lti_deployment { association(:lti_deployment) }
+    service_type { 'namesroles' }
```

Comment:
I think it's better to move this attribute into a child factory (so `:lti_service` becomes "abstract")